libs/ledger-live-common/src/families/cosmos/js-prepareTransaction.ts

```diff
@@ -12,36 +11,15 @@ import {
 import { getEnv } from "../../env";
 import { log } from "@ledgerhq/logs";
 
-export const calculateFees: CacheRes<
-  Array<{
-    account: Account;
-    transaction: Transaction;
-  }>,
-  {
-    estimatedFees: BigNumber;
-    estimatedGas: BigNumber;
-  }
-> = makeLRUCache(
-  async ({
-    account,
-    transaction,
-  }): Promise<{
-    estimatedFees: BigNumber;
-    estimatedGas: BigNumber;
-  }> => {
-    return await getEstimatedFees(account as CosmosAccount, transaction);
-  },
-  ({ account, transaction }) =>
-    `${account.id}_${account.currency.id}_${transaction.amount.toString()}_${
-      transaction.recipient
-    }_${String(transaction.useAllAmount)}_${transaction.mode}_${
-      transaction.validators
-        ? transaction.validators.map((v) => v.address).join("-")
-        : ""
-    }_${transaction.memo ? transaction.memo.toString() : ""}_${
-      transaction.sourceValidator ? transaction.sourceValidator : ""
-    }`
-);
+export const calculateFees = async ({
+  account,
+  transaction,
+}): Promise<{
+  estimatedFees: BigNumber;
+  estimatedGas: BigNumber;
+}> => {
+  return await getEstimatedFees(account as CosmosAccount, transaction);
+};
```

Comment:
`calculateFees` is now redundant, you can remove it and directly call `getEstimatedFees`

Reply:
Done